justfile

```
@@ -18,6 +21,14 @@ check:
 # Lint everything.
 lint:
```

```suggestion
lint: lint-verify lint-integration-tests
```

The other tests need to be listed as dependencies. It doesn't run as it currently is and I had to look up why: If you list them below it's run in the shell so would need to be `just lint-verify`.